packages/account-v2/src/modules/scripts/webpack-entries.json

```diff
@@ -0,0 +1,9 @@
+{
+  "index": "./src/index.ts",
+  "AddressFields": "./src/AddressFields/AddressFields.tsx",
+  "IDVForm": "./src/IDVForm/IDVForm.tsx",
+  "IDVService": "./src/IDVService/IDVService.tsx",
+  "ManualUpload": "./src/ManualUpload/ManualUpload.tsx",
+  "Onfido": "./src/Onfido/OnfidoContainer.tsx",
+  "POAForm": "./src/POAForm/POAFormContainer.tsx"
+}
```

Comment:
**NIT**: Please run prettier

Reply:
this file is generated from webpack-entries.js, won't make sense to re-sort this.

Reply:
Added this file to .gitignore.